Minor Edits & Corrections to Create Message (multipart/form-data) (#706)
discordapp/discord-api-docs: docs/resources/Channel.md

| content | string | the message contents (up to 2000 characters) | true |
| nonce | snowflake | a nonce that can be used for optimistic message sending | false |
| tts | boolean | true if this is a TTS message | false |

> | file | file contents | the contents of the file being sent | when sending `multipart/form-data` at least one of content, file, embeds must be present. |

Reviewer: ```suggestion
| file | file contents | the contents of the file being sent | `multipart/form-data` only |
```

It's better to include this requirement somewhere else, like in the endpoint description, as this requirement also applies to a JSON payload (just with no file uploads).

| embed | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object | embedded `rich` content | false |

> | payload_json | string | JSON encoded body used in place of the `embed` field | `multipart/form-data` only

Reviewer: ```suggestion
| payload_json | string | JSON encoded body of the additional params | `multipart/form-data` only
```

>info
>For the embed object, you can set every field except `type` (it will be `rich` regardless of if you try to set it), `provider`, `video`, and any `height`, `width`, or `proxy_url` values for images.

>info

> >`payload_json` must be of the format `{ embed: [embed](#DOCS_RESOURCES_CHANNEL/embed-object) }`, not simply the embed object.

Reviewer: Can be removed with the above suggestion, but must at least be updated as you can send any of the other params too, not only `embed`. You also need to update the warning above the table (`url-encoded JSON body` -> `JSON-encoded body`).
